src/alert.cpp

```diff
@@ -244,8 +246,22 @@ bool CAlert::ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThre
     return true;
 }
 
```

Comment:
If I recall correctly, the GUI will only show one alert at a time (the highest priority alert).

So using one static CAlert instance with a zero hash for all internally-generated alerts is the Simplest Thing That Will Work, and should work just fine.

Tom or Jonathan: you could do a quick test by adding some code in init.cpp that calls CAlert::NotifyInternal after a little bit.  Something like:
```
scheduler.scheduleFromNow(boost::bind(CAlert::NotifyInternal, "One Minute Warning!"), 60);
scheduler.scheduleFromNow(boost::bind(CAlert::NotifyInternal, "Two Minute Warning!"), 120);
```